src/main/java/com/cedrus/patpetillo/springkafkapingpong/stream/TopologyProvider.java

```diff
@@ -50,20 +47,17 @@ public Topology getTopology(PingPongTarget pingPongTarget) {
         return builder.build();
     }
 
-    private KStream<String, String> getPingPongStream(KStream<String, String> incomingStream, PingPongTarget pingPongTarget) {
+    private KStream<String, String> getPingPongStream(KStream<String, String> incomingStream, PingPongTeam pingPongTeam) {
         final KStream<String, String> pingPongStream = incomingStream
-                // Ensure we aren't processing volleyCount stream
-                .filter((key, value) -> value.contains("pingPongTarget"))
                 .filter((key, value) -> {
```

Comment:
Please move this lambda into a private method.  Also, why filter instead of branch?